include/ignition/physics/ConstructEmpty.hh

```diff
@@ -26,13 +26,18 @@ namespace ignition {
 namespace physics {
 
 /////////////////////////////////////////////////
+/// \brief This feature constructs an empty World entity and return its pointer
+/// from the current physics engine in use.
 class ConstructEmptyWorldFeature : public virtual Feature
 {
   public: template <typename PolicyT, typename FeaturesT>
   class Engine : public virtual Feature::Engine<PolicyT, FeaturesT>
   {
     public: using WorldPtrType = WorldPtr<PolicyT, FeaturesT>;
 
+    /// \brief Construct an empty World and attach a given name to it.
```

Comment:
I think it'd make sense to say either `World entity` or just `world`, otherwise the capitalization would look confusing.